joomla/joomla-cms
# com_languages installed view, implement searchtools and ordering

#### Description

This PR adds, for consistency, as it exists in so many other views in the backoffice, the search tools (filte, order, etc) to com_languages installed view.

Other small code changes/improvements where made.

Please notice that now, since the client filter is incorporated in the "installed" view (just like in modules view), the two options in the left bar for this doesn't make much sense.
I will make another PR to propose a change to that.
###### Before PR

![com_languages-installed-beforepr](https://cloud.githubusercontent.com/assets/9630530/12964050/eae1566c-d045-11e5-8303-b3812e2b9c55.png)
###### After PR

![image](https://cloud.githubusercontent.com/assets/9630530/12964106/2d4a56ac-d046-11e5-949e-b05c57ea0853.png)
#### How to test
1. Install latest staging and apply this patch
2. Go to Extensions -> Languages -> Installed - Site and Installed - Administrator
3. Check if everything is working fine.
